servlet/_index.md

```diff
@@ -1,7 +1,173 @@
 ---
 title: "Jakarta Servlet"
 summary: "Jakarta Servlet defines a server-side API for handling HTTP requests and responses."
+description: "Find all releases of the Jakarta Servlet specification which offers a number of APIs that can be used to create web based user interfaces."
```

Comment:
It isn't a number of AIs, it is one API - the Servlet API.

Why the focus on the user interface? The Servlet API supports more than just UI.